src/Persistence/UI.php

```diff
@@ -126,8 +126,8 @@ public function _typecastLoadField(\atk4\data\Field $f, $value)
         case 'date':
         case 'datetime':
         case 'time':
-            $dt_class = isset($f->dateTimeClass) ? $f->dateTimeClass : 'DateTime';
-            $tz_class = isset($f->dateTimeZoneClass) ? $f->dateTimeZoneClass : 'DateTimeZone';
+            $dt_class = $f->dateTimeClass ?? \DateTime::class;
+            $tz_class = $f->dateTimeZoneClass ?? \DateTimeZone::class;
```

Comment:
this change behaviour if `dateTimeClass` was defined as `false` or empty string for example.

Reply:
The same as above, `??` should be equal to `isset`. (in oposite to `?:` or `!empty`)

Reply:
false should still work, but it should also solve issues with implied namespaces..

Reply:
> implied namespaces

There should not be any, can you provide more details? (but same here, this PR produces exactly the same string as the hardcoded one before ;-))